src/test/rustdoc/auxiliary/external-macro-src.rs

```diff
@@ -0,0 +1,15 @@
+// compile-flags:--remap-path-prefix={{src-base}}=/does-not-exist
+
+#![doc(html_root_url = "https://example.com/")]
```

Comment:
Can you add a test that uses `thread_local!` from libstd and checks that the resulting docs have `doc.rust-lang.org` URLs? Since that was my smallest example for #70757 (but I guess I left it at the end of this comment: https://github.com/rust-lang/rust/issues/70025#issuecomment-607697742).

The libstd URLs should be already valid today, so it'd be a good test.

Reply:
I've now added a test using `thread_local!`.